packages/hdwallet-core/src/solana.ts

```diff
@@ -21,9 +21,9 @@ export interface SolanaTxInstruction {
 export interface SolanaSignTx {
   addressNList: BIP32Path;
   /** to is the destination pubkey for the transfer */
-  to: string;
+  to?: string;
   /** value is the amount to transfer in micro lamports*/
-  value: string;
+  value?: string;
```

Comment:
preferably-blocking: conceptually fine if not required to interact with PDAs, but not sure how I feel about this being in the main interface, introduces a lot of risks for all other Txs. 

Perhaps a program variant of this interface would make more sense instead?

Reply:
Ah sorry I should have convert this to a draft, as seen with @kaladinlight we already zero out those parameters to comply with SPL usages, we might be able to do the same for any other program usage

I need to verify it works properly before closing this PR 🙏